### PR TITLE
 Document what each check does and the types of errors it might return

### PR DESCRIPTION
**Pull request by rcritten**
Provide some guidance on what can cause specific errors and include a sample of what one might look like to help drive search requests to this file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -267,3 +267,655 @@ to do that manually:
...
+        "range_start": 1000,
+        "range_max": 199999,
+        "next_start": 0,
+        "next_max": 0,
+      }
+    }
+
+## ipahealthcheck.ipa.files
+
+These checks verify the owner and mode of files installed or configured by IPA. There are many permuations of file permissions and ownership that may be valid and continue to work. This reports on the expected values in a fresh IPA installation. Deviations are reported at the WARNING level.
```
Comment (fcami) [inline README.md]: permutations (missing 't')

File: README.md [hunk 2]
```diff
@@ -267,3 +267,655 @@ to do that manually:
...
+
+Check on available disk space. Running low can cause issues with logging, execution and backups.
+
+### FileSystemSpaceCheck
+
+Both a percentage and raw minimum values are tested.
+
+It is possible there is some overlap depending on mount points.
+
+The minimum free is 20 and is currently hardcoded.
```
Comment (fcami) [inline README.md]: minimum free percentage (missing: 'percentage')

File: README.md [hunk 3]
```diff
@@ -267,3 +267,655 @@ to do that manually:
...
     # echo "[default]" > /etc/ipahealthcheck/ipahealthcheck.conf
+
+# Understanding the results
+
+Here is some basic guidance on what a non-SUCCESS message from a check means. How to fix any particular result is heavily dependent on the error(s) discovered and their context. A single failure may be detected by multiple checks.
+
+## ipahealthcheck.dogtag.ca
+
+### DogtagCertsConfigCheck
+Compares the value of the CA (and KRA if installed) certificates with the value found in CS.cfg. If they don't match then the CA will likley fail to start.
```
Comment (fcami) [inline README.md]: likely (missing 'e')

File: README.md [hunk 4]
```diff
@@ -267,3 +267,655 @@ to do that manually:
...
+      "check": "IPACertTracking",
+      "result": "WARNING",
+      "kw": {
+        "key": 1234,
+        "msg": "Unknown certmonger id 1234'
+      }
+    }
+
+### IPACertNSSTrust
+The trust for certificates stored in NSS databases is compared against a known good.
```
Comment (fcami) [inline README.md]: maybe:
known good +state

Comment (rcritten) [reply README.md]: done.

File: README.md (hunk 1 above)
Comment (rcritten) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (rcritten) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (rcritten) [reply README.md]: done

**Comment by fcami** [comment]
All requested changes accounted for. ACK.

**Pull request merged**